splash/browser_tab.py

```diff
@@ -528,6 +528,13 @@ def http_get(self, url, callback, headers=None, follow_redirects=True):
             follow_redirects=follow_redirects
         )
 
+    def http_post(self, url, callback, headers=None, follow_redirects=True, body=None):
+        self.http_client.post(url,
+                              callback=callback,
+                              headers=headers,
+                              follow_redirects=True,
```

Comment:
why is there `follow_redirects` argument if it is not passed to self.http_client?